cmd/apps/registry_ingress_app.go

```diff
@@ -205,3 +213,48 @@ spec:
     - http01:
         ingress:
           class: {{.IngressClass}}`
+
+// Ingress in networking.k8s.io/v1 was added in k8s 1.19+
+// this includes the pathType change added in 1.18
+var registryIngressNetworkingYamlTemplate = `
```

Comment:
Reading this, I wonder whether you concluded that is it easier for us to maintain two templates and phase out the older one in due course than use more conditionals?

Reply:
I picked separate templates because i thought it would be more maintainable for myself and other people. It was just easier to read, the `if` statements in the template are a bit messy and as you point out, we can probably just _drop_ the the old template in a year or so and deleting a template is easier than undoing the if statements, imo.